interactive-repl.md

```diff
@@ -39,11 +39,13 @@ on the toolbar:
  
```

Comment:
```md
you can press **ESCAPE**, or click on the **Interrupt R **button  on the toolbar
```

Note the position of asterisks around `Interrupt R`. This breaks formatting in the output.